api/docs/release.dox

```diff
@@ -141,6 +141,10 @@ changes:
    member fields of classes are now following a consistent style with
    an underscore suffix.  References to renamed fields will need to be
    updated.
+ - Changed the signature of the dr_flush_region() instrument API, in order to
```

Comment:
Generally we have avoided such breaks, esp for binary compatibility breaks like this (new DR with old client will use garbage callback value and probably crash).  We instead add an _ex version, so `dr_flush_region_ex()` (or we use a struct with a size for graceful field appending).

Reply:
I added `dr_flush_region_ex` with the additional args, but didn't use a struct as I don't think new args would be added frequently.